moveit_planners/stomp/industrial_collision_detection/industrial_collision_detection_plugin_description.xml

```diff
@@ -0,0 +1,7 @@
+<library path="lib/libindustrial_collision_detection_plugin">
+  <class name="IndustrialFCL" type="collision_detection::IndustrialFCLPluginLoader" base_class_type="collision_detection::CollisionPlugin">
+    <description>
+      A implementation of an improved version of moveit's fcl collision detection.
```

Comment:
That would be the right approach, we could create another PR to bring such improvements into the FCL plugins.